src/solvers/qbf/qbf_bdd_core.cpp

```diff
@@ -212,13 +225,13 @@ propt::resultt qbf_bdd_coret::prop_solve()
         matrix->nodeCount() << " nodes" << std::endl;
       #endif
 
-      BDD* model = new BDD();
+      BDD *model=new BDD();
       const BDD &varbdd=*bdd_variable_map[var];
-      *model = matrix->AndAbstract(varbdd.Xnor(bdd_manager->bddOne()),
+      *model=matrix->AndAbstract(varbdd.Xnor(bdd_manager->bddOne()),
                                    varbdd);
```

Comment:
I've created a PR to make the linter spot this error: https://github.com/diffblue/cbmc/pull/415